### PR TITLE
5.4.10-8

### PR DESCRIPTION
**Pull request by RoeyPrat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,153 +1,146 @@
...
+- [Advanced Topics](docs/topics.md)
+- [Resource Specification Reference](docs/operator.md)
 
-Service Broker      -   `redislabs/service-broker:78_4b9b17f` or `redislabs/service-broker:78_4b9b17f.rhel7`
+## Quickstart Guide
 
+### Prerequisites
 
+- A minimum of 3 nodes which support the following requirements
+- A Kubernetes cluster (server) version of 1.8 or higher
```
Comment (RoeyPrat) [inline README.md]: 1.9?

File: README.md [hunk 2]
```diff
@@ -1,153 +1,146 @@
...
-    ```
+Redis Enterprise    -   `redislabs/redis:5.4.10-22` or `redislabs/redis:5.4.6-22b.rhel7-openshift`
 
-    For OpenShift deployments, create a new project (you can substitute `oc` for `kubectl` in the rest of these instructions):
+Operator            -   `redislabs/operator:5.4.10-8` or `redislabs/operator:5.4.6-8.rhel7`
 
-    ```
-    oc new-project my-project
-    ```
+Services Rigger     -   `redislabs/k8s-controller:5.4.10-8` or `redislabs/k8s-controller:5.4.10-8b.rhel7`
```
Comment (RoeyPrat) [inline README.md]: 5.4.10-8**b**?

File: README.md [hunk 3]
```diff
@@ -1,153 +1,146 @@
...
+```
 
-8)  Create A Redis Enterprise Cluster.  Choose the configuration relevant for you (see next section).  There are additional examples in the examples folder. Note that you need to specify an image tag if you'd like to pull a RHEL image.
+4. Deploy the OpenShift operator bundle:
 
-    ```
-    kubectl apply -f redis-enterprise-cluster.yaml
-    ```
+```bash
+kubectl apply -f openshift.bundle.yaml
```
Comment (RoeyPrat) [inline README.md]: oc apply

File: README.md (hunk 2 above)
Comment (MeniKatz) [reply README.md]: it is the image on RedHat site

Comment (amiramm) [reply README.md]: 8b is on RedHat connect which we needed to retag due to the labels issue

File: README.md (hunk 3 above)
Comment (amiramm) [reply README.md]: we can, but don't have to since we oc applied the scc.yaml separately.

**Pull request merged**